4337/src/utils/execution.ts

```diff
@@ -1,9 +1,7 @@
-import { Contract, Wallet, utils, BigNumber, BigNumberish, Signer, PopulatedTransaction } from 'ethers'
-import { TypedDataSigner } from '@ethersproject/abstract-signer'
-import { AddressZero } from '@ethersproject/constants'
+import { BigNumberish, Contract, ContractTransaction, Signer, TransactionResponse, Wallet, ethers } from 'ethers'
 
-export const TRANSACTION_TO_EXECUTE_SLOT = utils.keccak256(utils.toUtf8Bytes('eip4337diatomic.transaction_hash_to_execute'))
-export const SAFE_EIP4337_NONCE_SLOT = utils.keccak256(utils.toUtf8Bytes('eip4337diatomic.nonce'))
+export const TRANSACTION_TO_EXECUTE_SLOT = ethers.id('eip4337diatomic.transaction_hash_to_execute')
+export const SAFE_EIP4337_NONCE_SLOT = ethers.id('eip4337diatomic.nonce')
```

Comment:
I think these can be removed because they're coming from an older iteration of the module where the entrypoint didn't track the nonce, and we had to track validated transactions and nonce in the module.